### DISCUSSION TITLE
Microcks and developers

### DISCUSSION THREAD

**Discussion by yada** [microcks/microcks#901] [Announcements]
Hi all,
Latest articles on Microcks and developers see:
How Microcks fit and unify Inner and Outer Loops for cloud-native development
https://www.linkedin.com/pulse/how-microcks-fit-unify-inner-outer-loops-cloud-native-kheddache

If you would like concrete and technical use cases and examples of Microcks' capabilities to enhance developers' work using Microcks on laptops and the inner loop in action, please read the blog posts of [Laurent Broudoux](https://www.linkedin.com/in/laurentbroudoux?miniProfileUrn=urn%3Ali%3Afs_miniProfile%3AACoAAAIPBbsBvINQZpsEFVoO75QoZ5wnJTDeh98) "Mocking and contract-testing in your Inner Loop with Microcks", here:
[Part 1: Easy environment setup](https://medium.com/itnext/mocking-and-contract-testing-in-your-inner-loop-with-microcks-part-1-easy-environment-setup-dcd0f4355231)
[Part 2: Unit testing with Testcontainers](https://medium.com/itnext/mocking-and-contract-testing-in-your-inner-loop-with-microcks-part-2-unit-testing-with-860a86cb4b4c)
[Part 3: Quarkus Devservice FTW](https://medium.com/@lbroudoux/mocking-and-contract-testing-in-your-inner-loop-with-microcks-part-3-quarkus-devservice-ftw-a14b807737be)

Regards,
Yacine

**Comment by lbroudoux** [comment]
Do not hesitate to share - here or on Medium posts - your thoughts and feelings on those use cases!
Also, we're looking for new contributors/maintainers for the different Testcontainers libraries or IDE-related configuration templates. Jump onboard and say hi! if interested 🚀
